### PR TITLE
Change default password for Grafana and update Grafana documentation

### PR DESCRIPTION
**Pull request by erzetpe**

### CODE REVIEW

File: docs/home/howto/MONITORING.md [hunk 1]
```diff
@@ -12,9 +12,22 @@ TODO
...
+
+```yaml
+kind: configuration/grafana
+specification:
+  ...
+  # Variables correspond to ones in grafana.ini configuration file
+  # Security
+  grafana_security:
+    admin_user: admin
+    admin_password: "YOUR_PASSWORD"
```
Comment (rafzei) [inline docs/home/howto/MONITORING.md:26]:
```suggestion
    admin_password: "PLEASE_CHANGE_THIS_PASSWORD"
```

File: docs/home/howto/MONITORING.md [hunk 2]
```diff
@@ -12,9 +12,22 @@ TODO
 
 ## How to setup default admin password and user in Grafana
 
-To use other default user than admin/admin created automatically after Grafana installation please follow official Grafana 
-documentation accessible at https://grafana.com/docs/grafana/latest/installation/configuration/#security address. You can also find
-there other useful configuration options for Grafana security.
+Prior to setup Grafana please setup in your configuration yaml new password and/or name for your admin user. If not dafault
+admin user will be used with default password PLEASE_CHANGE_THIS_PASSWORD.
```
Comment (rafzei) [inline docs/home/howto/MONITORING.md:16]:
```suggestion
"admin" user will be used with the default password "PLEASE_CHANGE_THIS_PASSWORD".
```

**Review by rafzei — CHANGES_REQUESTED** [review]

File: docs/home/howto/MONITORING.md (hunk 1 above)
Comment (erzetpe) [reply docs/home/howto/MONITORING.md:26]: Nope, this is also in other places in Epiphany. We are writing that this is a place where someone needs to pass his own password.

**Review by rafzei — APPROVED** [review]
Ok, then fine for me :)

File: docs/home/howto/MONITORING.md [hunk 3]
```diff
@@ -12,9 +12,22 @@ TODO
 
 ## How to setup default admin password and user in Grafana
 
-To use other default user than admin/admin created automatically after Grafana installation please follow official Grafana 
-documentation accessible at https://grafana.com/docs/grafana/latest/installation/configuration/#security address. You can also find
-there other useful configuration options for Grafana security.
+Prior to setup Grafana please setup in your configuration yaml new password and/or name for your admin user. If not dafault
```
Comment (przemyslavic) [inline docs/home/howto/MONITORING.md:15]: If not, d**e**fault

**Review by rafzei — APPROVED** [review]

**Review by przemyslavic — APPROVED** [review]

**Pull request merged**